src/CoreWCF.Http/tests/FaultContractTests.cs

```diff
@@ -249,7 +250,7 @@
 
                 try
                 {
-                    string response = channel.TwoWayAsync_Method(f).GetAwaiter().GetResult();
+                    string response = await channel.TwoWayAsync_Method(f);
                     Assert.Fail($"Error, Client received: {response}");
```

Comment:
An await was added to this method without making the method async. It's causing a compilation error.